python/src/LinearModelAnalysis_doc.i.in

```diff
@@ -0,0 +1,283 @@
+
+Returns
+-------
+tScores : :class:`~openturns.Point`
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LinearModelAnalysis::getCoefficientsPValues
+"Accessor to the coefficients of the p values.
+
+Returns
+-------
+pValues : :class:`~openturns.Point`
+"
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LinearModelAnalysis::getCoefficientsConfidenceInterval
+"Accessor to the confidence interval of level :math:`\alpha` for the coefficients
+of the linear expansion
+
+Returns
+-------
+confidenceInterval : :class:`~openturns.Interval`
+"
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LinearModelAnalysis::getRSquared
+"Accessor to the R-squared test.
+
+Returns
+-------
+rSquared : float
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LinearModelAnalysis::getAdjustedRSquared
+"Accessor to the Adjusted R-squared test.
+
+Returns
+-------
+adjustedRSquared : float
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LinearModelAnalysis::getFisherScore
+"Accessor to the Fisher test.
+
+Returns
+-------
+fisherScore : float
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LinearModelAnalysis::getFisherPValue
+"Accessor to the Fisher p value.
+
+Returns
+-------
+fisherPValue : float
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LinearModelAnalysis::getNormalityTestResultChiSquared
+"Performs the Chi-Square statistical test to check Gaussian assumption of the model
+
+Returns
+-------
+testResult : :class:`~openturns.TestResult`
+    Test result class.
+
+Notes
+-----
+The Chi-Square test is a goodness of fit test which objective is to check the
+normality assumption (null hypothesis) of residuals (and thus the model).
+
+Usually, Chi-Square test applies for discrete distributions. Here we rely on
+the :class:`~openturns.FittingTest_ChiSquared` to check the normality.
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LinearModelAnalysis::getNormalityTestResultKolmogorovSmirnov
+"Performs Kolmogorov test to check Gaussian assumption of the model (null hypothesis).
+It is accepted when p-value is low.
+
+Returns
+-------
+testResult : :class:`~openturns.TestResult`
+    Test result class.
+
+
+Notes
+-----
+We check that the residual is Gaussian thanks to :class:`~openturns.FittingTest::Kolmogorov`."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LinearModelAnalysis::getNormalityTestResultAndersonDarling
+"Performs Anderson-Darling test to check Gaussian assumption of the model (null hypothesis).
+It is accepted when p-value is low.
```

Comment:
You will get pep warnings: may need a shorter one-line title, then a blank line, with imperative form (Perform without the s).